activerecord-jdbc-adapter.gemspec

```diff
@@ -37,5 +37,7 @@ Gem::Specification.new do |gem|
   #gem.add_development_dependency 'mocha', '~> 0.13.1'
 
   gem.rdoc_options = ["--main", "README.md", "-SHN", "-f", "darkfish"]
+
+  gem.extensions << 'Rakefile' if File.exist?('.git')
```

Comment:
do not like where this is going - we could easily leave such piece in for the released gem, which would be bad